packages/hooks/src/usePlatformDemoAccount.ts

```diff
@@ -0,0 +1,11 @@
+import { useStore } from '@deriv/stores';
+
+const usePlatformDemoAccount = () => {
+    const { client } = useStore();
+    const { accounts } = client;
+    const account_list = Object.keys(accounts).map(loginid => accounts[loginid]);
+
+    return account_list.find(account => account.is_virtual);
```

Comment:
@shayan-deriv Same here, If you like we can keep it in a helper variable first and then return it.